public/js/draft.js

```diff
@@ -64,72 +64,66 @@ function arrayRotate(arr, reverse) {
   return arr;
 }
 
+function arrayIsSubset(needles, haystack) {
+  return needles.every(x => haystack.includes(x));
+}
+
+const fetchLands = [
+  'Arid Mesa',
+  'Bloodstained Mire',
+  'Flooded Strand',
+  'Marsh Flats',
+  'Misty Rainforest',
+  'Polluted Delta',
+  'Scalding Tarn',
+  'Verdant Catacombs',
+  'Windswept Heath',
+  'Wooded Foothills',
```

Comment:
```suggestion
  'Wooded Foothills',
  'Bant Panorama',
  'Esper Panorama',
  'Grixis Panorama',
  'Jund Panorama',
  'Naya Panorama',
```
Do you want to include the 3 colour fetch panoramas?  From https://mtg.gamepedia.com/Fetch_land

Reply:
I don't think so. I think it might be worth to include prismatic vista though.

Reply:
My assumption was people would have vista in the colorless section and therefore it wouldn't be a problem. This special case is specifically for one-on-color fetches, not for others.